### PR TITLE
PRSice EB file created

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PRSice/PRSice-2.2.12-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    ('CMake', '3.15.3'),
+    ('binutils', '2.32'),
+    ('Eigen', '3.3.7', '', True),
+]
+
+files_to_copy = [(['bin/PRSice'], 'bin'), "README.md", "LICENSE"]
+
+sanity_check_paths = {
+    'files': ['bin/PRSice'],
+    'dirs': ['bin'],
```
Comment (boegel) [inline easybuild/easyconfigs/p/PRSice/PRSice-2.2.12-GCCcore-8.3.0.eb]: @sassy-crick Also checking for a non-empty `bin` directory when you're already checking for `bin/PRSice` is a bit silly, so let's trim this down to:

```python
sanity_check_paths = {
    'files': ['bin/PRSice'],
    'dirs': [],
}

File: easybuild/easyconfigs/p/PRSice/PRSice-2.2.12-GCCcore-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
```
Comment (boegel) [inline easybuild/easyconfigs/p/PRSice/PRSice-2.2.12-GCCcore-8.3.0.eb]: Let's drop the duplicate empty line, and add a sanity check command instead to ensure the binary works:

```python
sanity_check_commands = ["PRSice --help"]

Comment (sassy-crick) [reply easybuild/easyconfigs/p/PRSice/PRSice-2.2.12-GCCcore-8.3.0.eb]: Thanks @boegel ! That makes more sense to me as well.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3120.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/4314b23478a6922d46c4b193dfd238d4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/91e2c97656e13c6f49c33a5026404f5e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d4991b666112f0e09f83d1b4e87b3a8a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**